README.md

```diff
@@ -110,6 +110,7 @@ optional arguments:
   --master NODE         Node where to deploy the master (required for new
```

Comment:
I guess ``[-u]`` is missing above

Reply:
hi! i do not see/match what is missing.? short and long option in readme. https://github.com/mlfmonde/cluster_cli/pull/12/commits/c30f7a95d0df25634aad822968aec1d6befda705

Reply:
I couldn't comment but few lines above: https://github.com/mlfmonde/cluster_cli/blame/master/README.md#L99 should be updated, insn't it ?

Reply:
ok, got it !

Reply:
done